### PR TITLE
HDDS-8361. Useless object in TestOzoneBlockTokenIdentifier

### PR DESCRIPTION
**Pull request by adoroszlai**
## What changes were proposed in this pull request?

Fix findbugs violation in `TestOzoneBlockTokenIdentifier`, which interestingly only appeared in #4506 without any change in that file.

Few other minor fixes in the same file.

https://issues.apache.org/jira/browse/HDDS-8361

## How was this patch tested?

Findbugs and unit test.

https://github.com/adoroszlai/hadoop-ozone/actions/runs/4594024215

### CODE REVIEW

**Comment by kerneltime** [comment]
cc @hemantk-12 can you please take a look?

File: hadoop-hdds/common/src/test/java/org/apache/hadoop/hdds/security/token/TestOzoneBlockTokenIdentifier.java [hunk 1]
```diff
@@ -260,15 +259,15 @@ public void testAsymmetricTokenPerf() throws NoSuchAlgorithmException,
...
+  void testSymmetricTokenPerf() {
     String hmacSHA1 = "HmacSHA1";
     String hmacSHA256 = "HmacSHA256";
 
     testSymmetricTokenPerfHelper(hmacSHA1, 64);
     testSymmetricTokenPerfHelper(hmacSHA256, 1024);
   }
 
-  public void testSymmetricTokenPerfHelper(String hmacAlgorithm, int keyLen) {
+  private void testSymmetricTokenPerfHelper(String hmacAlgorithm, int keyLen) {
```
Comment (ChenSammi) [inline hadoop-hdds/common/src/test/java/org/apache/hadoop/hdds/security/token/TestOzoneBlockTokenIdentifier.java:270]: @adoroszlai , thanks for fix the issue.  
Does the scope change from public to private solve the problem?

File: hadoop-hdds/hadoop-dependency-test/pom.xml [hunk 2]
```diff
@@ -54,6 +54,10 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>commons-codec</groupId>
+      <artifactId>commons-codec</artifactId>
+    </dependency>
```
Comment (ChenSammi) [inline hadoop-hdds/hadoop-dependency-test/pom.xml:60]: Is this change related?

File: hadoop-hdds/common/src/test/java/org/apache/hadoop/hdds/security/token/TestOzoneBlockTokenIdentifier.java (hunk 1 above)
Comment (adoroszlai) [reply hadoop-hdds/common/src/test/java/org/apache/hadoop/hdds/security/token/TestOzoneBlockTokenIdentifier.java:270]: No, the problem is fixed by actually using the "useless object" here:

```
    LOG.info("Average token sign time with {}({} symmetric key) is {} ns",
        hmacAlgorithm, keyLen, duration / tokenPasswordSym.size());
```

File: hadoop-hdds/hadoop-dependency-test/pom.xml (hunk 2 above)
Comment (adoroszlai) [reply hadoop-hdds/hadoop-dependency-test/pom.xml:60]: `OzoneBlockTokenIdentifier` is in `hadoop-hdds/common`, so I'm moving the test to the same submodule.  This dependency is required at runtime for Hadoop code invoked by the test.

```
[ERROR] Tests run: 4, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.506 s <<< FAILURE! - in org.apache.hadoop.hdds.security.token.TestOzoneBlockTokenIdentifier
[ERROR] org.apache.hadoop.hdds.security.token.TestOzoneBlockTokenIdentifier.testTokenSerialization  Time elapsed: 0.04 s  <<< ERROR!
java.lang.NoClassDefFoundError: org/apache/commons/codec/binary/Base64
	at org.apache.hadoop.security.token.Token.encodeWritable(Token.java:342)
	at org.apache.hadoop.security.token.Token.encodeToUrlString(Token.java:373)
	at org.apache.hadoop.hdds.security.token.TestOzoneBlockTokenIdentifier.testTokenSerialization(TestOzoneBlockTokenIdentifier.java:169)
```

**Comment by ChenSammi** [comment]
The patch LGTM +1.

**Comment by adoroszlai** [comment]
Thanks @ChenSammi for reviewing and committing it.
